<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_QWidget</class>
 <widget class="QWidget" name="main_QWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="main_horizontalLayout">
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QWidget" name="settings_verticalWidget" native="true">
     <property name="maximumSize">
      <size>
       <width>275</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="settings_verticalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>289</height>
         </size>
        </property>
        <property name="title">
         <string>Parameters</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="4" column="3">
          <widget class="QDoubleSpinBox" name="ExtraSleepTime_doubleSpinBox">
           <property name="suffix">
            <string> ms</string>
           </property>
           <property name="decimals">
            <number>0</number>
           </property>
           <property name="singleStep">
            <double>0.000100000000000</double>
           </property>
          </widget>
         </item>
         <item row="5" column="3">
          <widget class="QPushButton" name="AutoLevel_PushButton">
           <property name="text">
            <string>Auto Level</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QLabel" name="Vxy">
           <property name="text">
            <string>XY Velocity</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QPushButton" name="AutoFocus_PushButton">
           <property name="text">
            <string>Auto Focus</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="3">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Delta(z/y)</string>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QDoubleSpinBox" name="delta_zy_doubleSpinBox">
           <property name="suffix">
            <string/>
           </property>
           <property name="decimals">
            <number>4</number>
           </property>
           <property name="minimum">
            <double>-1.000000000000000</double>
           </property>
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.000100000000000</double>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QDoubleSpinBox" name="Vxy_doubleSpinBox">
           <property name="suffix">
            <string> um/sec</string>
           </property>
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="maximum">
            <double>100.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.001000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="3">
          <widget class="QLabel" name="res_mode_label">
           <property name="text">
            <string>Delta(z/x)</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="camera_index_label_2">
           <property name="text">
            <string>Extra Sleep Time</string>
           </property>
          </widget>
         </item>
         <item row="6" column="3">
          <widget class="QPushButton" name="interrupt_pushButton">
           <property name="text">
            <string>Interrupt</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QDoubleSpinBox" name="delta_zx_doubleSpinBox">
           <property name="decimals">
            <number>4</number>
           </property>
           <property name="minimum">
            <double>-1.000000000000000</double>
           </property>
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.000100000000000</double>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QPushButton" name="start_pushButton">
           <property name="text">
            <string>Run</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Files</string>
        </property>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Job File Path:</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="Job_File_lineEdit">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>261</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="JobFileBrowseButton">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>20</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>100</y>
           <width>91</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Image File Path:</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="Img_File_lineEdit">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>130</y>
           <width>261</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="ImgFileBrowseButton">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>100</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="mode_line">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>80</y>
           <width>70</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>CheckBox</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="mode_pnt">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>80</y>
           <width>70</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>CheckBox</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton">
         <property name="geometry">
          <rect>
           <x>670</x>
           <y>150</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
        <widget class="QGroupBox" name="exposure_GroupBox">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>180</y>
           <width>244</width>
           <height>55</height>
          </rect>
         </property>
         <property name="title">
          <string>exposure time</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="leftMargin">
           <number>9</number>
          </property>
          <property name="topMargin">
           <number>9</number>
          </property>
          <property name="rightMargin">
           <number>9</number>
          </property>
          <property name="bottomMargin">
           <number>9</number>
          </property>
          <item>
           <widget class="QSlider" name="exposure_horizontalSlider">
            <property name="maximum">
             <number>2</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="exp_value">
            <property name="maximum">
             <double>5.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
            <property name="value">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QGroupBox" name="gain_horizontalGroupBox">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>235</y>
           <width>244</width>
           <height>55</height>
          </rect>
         </property>
         <property name="title">
          <string>gain</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QSlider" name="gain_horizontalSlider">
            <property name="minimum">
             <number>100</number>
            </property>
            <property name="maximum">
             <number>500</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="gain_doubleSpinBox">
            <property name="decimals">
             <number>0</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QCheckBox" name="live_checkBox">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>160</y>
           <width>48</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Live</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
